src/cellfinder_core/detect/filters/volume/ball_filter.py

```diff
@@ -187,30 +215,60 @@ def _cube_overlaps(
 
 @jit(nopython=True)
 def _is_tile_to_check(
-    x, y, middle_z, tile_step_width, tile_step_height, good_tiles_mask
+    x: int,
+    y: int,
+    middle_z: int,
+    tile_step_width: int,
+    tile_step_height: int,
+    good_tiles_mask: np.ndarray,
 ):  # Highly optimised because most time critical
+    """
+    Check if the tile containing pixel (x, y) is a tile that needs checking.
+    """
     x_in_mask = x // tile_step_width  # TEST: test bounds (-1 range)
     y_in_mask = y // tile_step_height  # TEST: test bounds (-1 range)
     return good_tiles_mask[x_in_mask, y_in_mask, middle_z]
 
 
 @jit(nopython=True)
 def _walk(
-    max_height,
-    max_width,
-    tile_step_width,
-    tile_step_height,
-    good_tiles_mask,
-    volume,
-    kernel,
-    ball_z_size,
-    ball_radius,
-    middle_z,
-    overlap_threshold,
-    THRESHOLD_VALUE,
-    SOMA_CENTRE_VALUE,
-):
+    max_height: int,
+    max_width: int,
+    tile_step_width: int,
+    tile_step_height: int,
+    good_tiles_mask: np.ndarray,
+    volume: np.ndarray,
+    kernel: np.ndarray,
+    ball_radius: int,
+    middle_z: int,
+    overlap_threshold: float,
+    THRESHOLD_VALUE: int,
+    SOMA_CENTRE_VALUE: int,
+) -> None:
     """
+    Scan through *volume*, and mark pixels where there are enough surrounding
+    pixels with high enough intensity.
+
+    The surrounding area is defined by the *kernel*.
+
+    Parameters
+    ----------
+    max_height, max_width :
+        Maximum offsets for the ball filter.
+    good_tiles_mask :
```

Comment:
In line with #114 we should rename this to `brain_tiles_mask` or `bright_tiles_mask`.

Reply:
👍 done